### PR TITLE
Create sc-SE-session.md

### PR DESCRIPTION
**Pull request by AliRKat**
few parts might need to update, waiting your opinions

### CODE REVIEW

**Comment by AliRKat** [comment]
[[SDK-Doc] Session scenario tests added in GitHub](https://trello.com/c/6UbebWsw/1878-sdk-doc-session-scenario-tests-added-in-github)

File: test_scenarios/sc-SE-session.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## 201_CNR_M
+        Same as 200_CR_CG_M except:
+        1. 4 reqs instead of 5 (no consent status req)
+
+## 202_CR_CNG_M
+        Same as 200_CR_CG_M except:
+        1. No requests generated
+
+## 203_CR_CG_M_id_change
+        endSession();
```
Comment (arifBurakDemiray) [inline test_scenarios/sc-SE-session.md:46]: tabbings are seeming like 4 please do it 2 afaik it is the way for doc

**Review by arifBurakDemiray — APPROVED** [review]

File: test_scenarios/sc-SE-session.md (hunk 1 above)
Comment (AliRKat) [reply test_scenarios/sc-SE-session.md]: thank you, fixed tabbings

**Review by turtledreams — CHANGES_REQUESTED** [review]
can you check the current flutter tests for these and update it accordingly.  also take the current observations from the notes.md of those flutter tests and add below

**Comment by turtledreams** [comment]
@AliRKat the more this waits, scarier it will get for you

**Review by turtledreams — APPROVED** [review]

**Pull request merged**